README.md

```diff
@@ -39,22 +39,22 @@ const WebSdkApi = require("@holo-host/web-sdk");
 const holo = new WebSdkApi(child);
 ```
 
-### `await WebSdkApi.connect({ chaperone_url, auth_form_customization }) -> WebSdkApi`
+### `await WebSdkApi.connect({ chaperone_url, authFormCustomization }) -> WebSdkApi`
```

Comment:
I'm gonna push a commit addressing snake_case in docs. This line for example should say `chaperoneUrl`

Reply:
Actually I'll make a PR on top of this one to avoid blocking this

Reply:
🙏

Reply:
🙏